person.rb

```diff
@@ -1,7 +1,7 @@
 require './nameable'
 
 class Person < Nameable
```

Comment:
- [ ] Amazing work was also done with the initialize method 👏💪. So to ensure that a person can rent a book, I kindly propose that you include a method called `add rental` in the `Book` class. 👌😊
An illustration of how to use the `add rental` method to implement the `has many` relationship between a `person` and a `rental` is provided below. :point_down: 

```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```